ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixFactory.java

```diff
@@ -33,5 +35,14 @@
   HelixManager getZKHelixManager(String clusterName, String instanceName, InstanceType instanceType, String zkAddr) {
     return HelixManagerFactory.getZKHelixManager(clusterName, instanceName, instanceType, zkAddr);
   }
+
+  /**
+   * Get a reference to a {@link HelixAdmin}
+   * @param zkAddr the address identifying the zk service to which this request is to be made.
+   * @return the constructed {@link HelixAdmin}.
+   */
+  HelixAdmin getHelixAdmin(String zkAddr) {
+    return new ZKHelixAdmin(zkAddr);
```

Comment:
Could you change this to `new HelixAdminFactory().getHelixAdmin(zkAddr)` - it does the same thing, but better to instantiate via the factory.

Reply:
Agree. Will address it soon. Thanks!